plugins/hls-cabal-plugin/test/Outline.hs

```diff
@@ -0,0 +1,86 @@
+        "cabal FieldLine outline test"
+        "fieldline.cabal"
+        [fieldLineDocumentSymbol],
+      testSymbols
+        "cabal Section outline test"
+        "section.cabal"
+        [sectionDocumentSymbol],
+      testSymbols
+        "cabal SectionArg outline test"
+        "sectionarg.cabal"
+        [sectionArgDocumentSymbol]
+    ]
+  where
+    fieldDocumentSymbol :: DocumentSymbol
+    fieldDocumentSymbol = (defDocumentSymbol (Range {_start = Position {_line = 0, _character = 0},
+                                                                 _end = Position {_line = 0, _character = 8}}))
+                                    { _name = "homepage",
+                                      _kind = T.SymbolKind_Field,
+                                      _children = Nothing
+                                    }
+    fieldLineDocumentSymbol :: DocumentSymbol
+    fieldLineDocumentSymbol = (defDocumentSymbol (Range {_start = Position {_line = 0, _character = 0},
+                                                                     _end = Position {_line = 0, _character = 13}}))
+                                    { _name = "cabal-version",
+                                      _kind = T.SymbolKind_Field,
+                                      _children = Nothing -- the values of fieldLine are removed from the outline
+                                    }
+    sectionDocumentSymbol :: DocumentSymbol
+    sectionDocumentSymbol = (defDocumentSymbol (Range {_start = Position {_line = 0, _character = 2},
+                                                                     _end = Position {_line = 0, _character = 15}}))
+                                    { _name = "build-depends",
+                                      _kind = T.SymbolKind_Field,
+                                      _children = Nothing -- the values of fieldLine are removed from the outline
+                                    }
+    sectionArgDocumentSymbol :: DocumentSymbol
+    sectionArgDocumentSymbol = (defDocumentSymbol (Range {_start = Position {_line = 0, _character = 2},
+                                                                     _end = Position {_line = 0, _character = 19}}))
+                                    { _name = "if os ( windows )",
+                                      _kind = T.SymbolKind_Object,
+                                      _children = Just $ [sectionArgChildrenDocumentSymbol]                                    }
+    sectionArgChildrenDocumentSymbol :: DocumentSymbol
+    sectionArgChildrenDocumentSymbol = (defDocumentSymbol (Range {_start = Position {_line = 1, _character = 4},
+                                                                     _end = Position {_line = 1, _character = 17}}))
+                                    { _name = "build-depends",
+                                      _kind = T.SymbolKind_Field,
+                                      _children = Nothing
+                                    }
```

Comment:
This formatting looks hard to read to me, could you format this section? Parts of the hls-cabal-plugin codebase are formatted with `fourmolu`.

Reply:
This formatting was accidental. Switched to the `fourmolu` with slight modifications.